site/content/in-dev/unreleased/configuring-polaris-for-production/configuring-gcs-cloud-storage-specific.md

```diff
@@ -0,0 +1,37 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: Configuring GCS Cloud Storage
+linkTitle: Configuring GCS Cloud Storage
+type: docs
+weight: 600
+---
+
+This page provides guidance for Configuring GCS Cloud Storage provider for use with Polaris. 
+It covers credential vending, IAM roles, ACL requirements, and best practices to ensure secure and reliable integration.
+
+#### GCS
+
+When using credential vending for Google Cloud Storage (GCS) with Apache Iceberg on
+Polaris, ensure that both IAM roles and HNS ACLs (if HNS is enabled) are properly configured. Even with the correct IAM
```

Comment:
Out of curious,  is HNS a must-have for credential vending? If yes, we have to clarify that.